app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1204,9 +1219,27 @@ BrowserWorld::StartFrame() {
     m.UpdateGazeModeState();
     m.UpdateControllers(relayoutWidgets);
     m.UpdateTrackedKeyboard();
-    if (m.inHeadLockMode) {
+    if (m.lockMode != LockMode::NO_LOCK) {
```

Comment:
What if we use the > operator instead of != ?

Reply:
For any particular reason? I prefer not to rely on the fact that the enums are integers under the hood